src/parser.ts

```diff
@@ -157,8 +159,11 @@ export class Parser {
 	 */
 	private setDelimiter(languageCode: string): void {
 		this.supportedLanguage = true;
-		
+		this.delimiters = [];
+
 		switch (languageCode) {
+			case "dart":
+				this.delimiters.push("///");
```

Comment:
Nope, sice there is no `break;` statement below line `166` execution continues with statements from next set of `cases`

Reply:
Ah I see, I missed that. I think it would be better to be explicit about this just in case the ordering of the cases changes. Missing a `break;` statement can be easily misconstrued as an error and not a deliberate choice